modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -296,6 +296,7 @@ export async function encryptNShare(
 
   const publicShare = Buffer.concat([
```

Comment:
is this change supported by hsm? I think this change will break key creation

Reply:
This is the public share, it's not sent to the HSM, at least not in this format. The HSM inputs are parameterized via object properties much the same way that the sdk-core tss module is.

Reply:
Unless I'm mistaken, the changes regarding sending the new v field would need to be made to WP.

Reply:
Currently this change will have v part of the public share (y) prop that will be sent to HSM from wp. if we want to exclude the v from y then we'll need to update WP.